movies-library/src/components/Counter.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+export class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      count: this.props.initialCount,
+    };
+  }
+
+  increment() {
+    this.setState({
```

Comment:
This could work not as expected. setState are asynch function. You need to pass function instead of object.

Reply:
I changed to functional component and used hooks